internal/http/web/user.go

```diff
@@ -327,7 +327,7 @@ func (h *Handler) getDashboard() http.HandlerFunc {
 
 		// Only show to accepted users
 		if a.Decision == models.DecisionAccepted {
-			if !a.AcceptedAt.IsZero() && time.Now().Sub(a.AcceptedAt) > models.RSVPExpiryTime {
+			if !a.AcceptedAt.IsZero() && (time.Now().UnixNano()/1000000) > models.RSVPExpiryDate {
```

Comment:
This makes every RSVP expire at the same time, is that intentional?

Reply:
Yeah based on the meeting notes it looks like all rsvps expire 12/18

Reply:
under comm -> RSVP it says 3 days, I'm assuming that's talking about a 3 day window to accept RSVPs, right? and then dec 18 is just the hard deadline

Reply:
oh actually thats not clear. I thought it was saying 3 days was how it used to be, cause the schedule says Dec 18 all rsvps due. I'll double check.

Reply:
If it is a 3 day window and a hard deadline of dec 18, I'll re-add the old condition

Reply:
Its a hard deadline for 12/18. The notes under comm were like inital thoughts

Reply:
Tho I'm not sure if we need the AcceptedAt column anymore with the hard column

Reply:
Yeah I guess we don't but it's not a big deal to keep it in

Reply:
I think if Comm doesn't utilize the AcceptedAt column, no rsvps would count as expired